### ISSUE TITLE
Train attacker found Loss=nan

### ISSUE THREAD

**Issue by Yanagi-kon** [siebeniris/ALGEN#2]
hello!

I discovered that the underlying T5LayerNorm module of the FLAN-T5 model in the code has been replaced with the standard PyTorch torch.nn.LayerNorm, which seems to be causing my training loss to consistently be NaN.

Looking forward to your answer!

**Comment by siebeniris** [comment]
Hi sorry for the very late response here. 

I did the following to accommodate apex errors from the server

`t5_modeling.T5LayerNorm = LayerNorm`

You could just comment out this line and start training, hope that works.
